src/imsosorry/uwuification.py

```diff
@@ -139,10 +162,13 @@ def uwuify(
     tilde_strength: float = 0.1,
 ) -> str:
     """Take a string and returns an uwuified version of it."""
-    text = text.lower()
-    text = word_replace(text)
-    text = nyaify(text)
-    text = char_replace(text)
-    text = stutter(text, stutter_strength)
-    text = emoji(text, emoji_strength)
-    return tildify(text, tilde_strength)
+    for letter in string.ascii_letters:
+        if letter in text:
+            return _uwuify(
+                text,
+                stutter_strength=stutter_strength,
+                emoji_strength=emoji_strength,
+                tilde_strength=tilde_strength,
+            )
+
+    return text
```

Comment:
We can use `isalpha` to exit early for invalid cases.
```suggestion
    if not text.isalpha():
        return text


    return _uwuify(
        text,
        stutter_strength=stutter_strength,
        emoji_strength=emoji_strength,
        tilde_strength=tilde_strength,
    )
```